sandbox/pom.xml

```diff
@@ -10,65 +10,54 @@
 	</parent>
 
     <artifactId>sandbox</artifactId>
-    <version>0.0.1-SNAPSHOT</version>
     <name>dcsa-conformance-sandbox</name>
     <description>DCSA Conformance Sandbox</description>
 
     <dependencies>
         <dependency>
             <groupId>org.dcsa.conformance</groupId>
             <artifactId>core</artifactId>
-            <version>0.0.1-SNAPSHOT</version>
         </dependency>
         <dependency>
             <groupId>org.dcsa.conformance</groupId>
             <artifactId>booking</artifactId>
-            <version>0.0.1-SNAPSHOT</version>
         </dependency>
         <dependency>
-            <groupId>org.dcsa.conformance</groupId>
-			<artifactId>ebl</artifactId>
-			<version>0.0.1-SNAPSHOT</version>
-		</dependency>
+          <groupId>org.dcsa.conformance</groupId>
```

Comment:
Could you adjust the indentation ? as the above pom files are indented properly, but rest of the below files are not.

Reply:
Yeah... I was in doubt to fix all those things as well. Shall I reformat all pom.xml files, after I got approvals (not messing up the diff, while reviewing)

Reply:
I approve it, please reformat the poms.